public/app/features/dashboard/api/pubDashApi.ts

```diff
@@ -0,0 +1,56 @@
+import { BaseQueryFn, createApi, retry } from '@reduxjs/toolkit/query/react';
+import { lastValueFrom } from 'rxjs';
+
+import { BackendSrvRequest, getBackendSrv } from '@grafana/runtime/src';
+import { notifyApp } from 'app/core/actions';
+import { createSuccessNotification } from 'app/core/copy/appNotification';
+import { PublicDashboard } from 'app/features/dashboard/components/ShareModal/SharePublicDashboard/SharePublicDashboardUtils';
```

Comment:
Do we want to use an absolute path here or should be relative?

Reply:
the best way is to use absolute and don't concatenate things like ../.../../